sequencer/src/persistence/fs.rs

```diff
@@ -623,8 +635,10 @@ impl SequencerPersistence for Persistence {
     }
     async fn append_quorum_proposal(
         &self,
-        proposal: &Proposal<SeqTypes, QuorumProposal<SeqTypes>>,
+        proposal: &Proposal<SeqTypes, QuorumProposal2<SeqTypes>>,
```

Comment:
ok we are storing QuorumProposal2. I think that existing QuorumProposal1 stored will not be deserialized into QuorumProposal2 so we need to think about that